src/Microsoft.ML.StaticPipe/SdcaStaticExtensions.cs

```diff
@@ -154,7 +154,7 @@ public static class SdcaStaticExtensions
             var rec = new TrainerEstimatorReconciler.BinaryClassifier(
                 (env, labelName, featuresName, weightsName) =>
                 {
-                    var trainer = new SdcaBinaryTrainer(env, labelName, featuresName, weightsName, l2Regularization, l1Threshold, numberOfIterations);
+                    var trainer = new SdcaCalibratedBinaryClassificationTrainer(env, labelName, featuresName, weightsName, l2Regularization, l1Threshold, numberOfIterations);
```

Comment:
>SdcaCalibratedBinaryClassificationTrainer [](start = 38, length = 41)

Did leave a note below, but i'd be ok dropping `Classificaiton` from all BinaryClassification and MulticlassClassification.

Reply:
note below .. am not getting you ?

---
In reply to: [264518672](https://github.com/dotnet/machinelearning/pull/2903#discussion_r264518672) [](ancestors = 264518672)